src/main/resources/raster-catalog-default.json

```diff
@@ -226,7 +222,7 @@
     },
     {
       "name":"umd_tree_cover_loss_from_fires",
-      "source_uri": "s3://gfw-data-lake/umd_tree_cover_loss_from_fires/v20230424/raster/epsg-4326/10/40000/year/geotiff/{tile_id}.tif"
+      "source_uri": "s3://gfw-data-lake/umd_tree_cover_loss_from_fires/v20240301/raster/epsg-4326/10/40000/year/geotiff/{tile_id}.tif"
```

Comment:
Did you want to switch to the usual pattern and use `{grid_size}/{row_count}`, rather than `10/40000` here?